SyslogNet.Client/Transport/SyslogTcpSender.cs

```diff
@@ -37,6 +38,11 @@ public SyslogTcpSender(string hostname, int port)
 
 		protected void Connect()
 		{

```

Comment:
I noticed this bug a moment ago, if a subclass calls `Connect()` twice in a row, this will leak resources because it's not calling `tcpClient.Dispose()`. So, as calling the function twice in a row is an error, better to throw an exception.